packages/terra-doc-template/src/NpmBadge.module.scss

```diff
@@ -0,0 +1,28 @@
+:local {
+  .badge {
+    color: #fff;
+    font-family: Verdana,DejaVu Sans,sans-serif;
+    font-size: 0.7857rem;
```

Comment:
Should we have been overriding font-size and font-family ?

Reply:
The normal npm badge also uses its own font size and family. This allows the non-badgen version of the badge to maintain consistency with the dynamic version.